build/rbac/get-helm-rbac.sh

```diff
@@ -1,13 +1,27 @@
 #!/usr/bin/env bash
 set -eEuox pipefail
 
-: ${HELM:=helm}
+: "${HELM:=helm}"
+
+# Whether to include Pod Security Policy (PSP) related resources in the RBAC output.
+# Empty string means DO include PSP resources. Any other value means do NOT include PSP resources.
+: "${DO_NOT_INCLUDE_POD_SECURITY_POLICY_RESOURCES:=""}"
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
-pushd "$SCRIPT_DIR"
+pushd "$SCRIPT_DIR" &>/dev/stderr
+
+options=(
+  --namespace rook-ceph
+  --set crds.enabled=false
+)
+if [[ -z "${DO_NOT_INCLUDE_POD_SECURITY_POLICY_RESOURCES}" ]]; then
+  options+=(--set pspEnable=true)
+else
+  options+=(--set pspEnable=false)
+fi
+
+echo "generating Helm template with options: ${options[*]}" &>/dev/stderr
 
-${HELM} template ../../cluster/charts/rook-ceph \
-                  --namespace rook-ceph \
-                  --set crds.enabled=false | ./keep-rbac-yaml.py > rbac.yaml
```

Comment:
This script no longer outputs to a file directly but uses stdout as its output. This allows the output to be used but sent to different places by `Makefile` and by `generate-rook-csv.sh`.

Reply:
Did you mean "use stderr as its output"?